klippy/extras/bed_mesh.py

```diff
@@ -1,6 +1,5 @@
 # Mesh Bed Leveling
 #
-# Copyright (C) 2018  Kevin O'Connor <kevin@koconnor.net>
```

Comment:
why remove this?

Reply:
He removed it [at Kevin's request](https://github.com/Klipper3d/klipper/pull/6461#discussion_r1458008900).

Reply:
sorry, thanks for clarifying